service/src/main/java/bio/terra/pipelines/db/entities/PipelineOutputDefinition.java

```diff
@@ -0,0 +1,25 @@
+package bio.terra.pipelines.db.entities;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@Table(name = "pipeline_output_definitions")
+public class PipelineOutputDefinition extends BasePipelineVariableDefinition {
+
+  @Column(name = "type", nullable = false)
+  private String type;
```

Comment:
i think this should be an enum thats share with `PipelineInputDefinition`.  so probalby rename `PipelineInputTypesEnum` to `PipelineypesEnum` and use them in both?  what do you think

Reply:
probably yes eventually, but for now since there's only one output type, I was keeping it as FILE and leaving it at that, because I didn't want to add FILE to `PipelineInputTypesEnum` and then have the FILE vs VCF discrepancy to leave or resolve in this PR. I'd prefer to make a new ticket to change `VCF` to `FILE` in `PipelineInputTypesEnum` (which would involve adding a field for the suffix to check for input validation) (and rename `PipelineInputTypesEnum` to `PipelineVariableTypesEnum`). thoughts?

Reply:
https://broadworkbench.atlassian.net/browse/TSPS-268